doc/api/v0.yaml

```diff
@@ -626,6 +691,37 @@ paths:
         '400':
           description: Message is malformed
 
+  /api/v0/messages:
+    post:
+      description: Post a batch of signed transactions
+      operationId: Messages
+      tags:
+        - fragment
+      requestBody:
+        description: An array of hex-encoded signed transactions
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: string
+                pattern: '[0-9a-f]+'
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                description: A list of fragment IDs
```

Comment:
ordered in the same order as in the input?

Reply:
Yes.

Reply:
document it then, please